go.mod

```diff
@@ -1,11 +1,20 @@
 module github.com/ipfs/go-ds-badger2
 
 require (
-	github.com/dgraph-io/badger/v2 v2.0.3
+	github.com/DataDog/zstd v1.4.5 // indirect
+	github.com/dgraph-io/badger/v2 v2.2007.2
```

Comment:
CI is failing due to badger/v2 v2.2007.2 being unable to build on x86. https://discuss.dgraph.io/t/badger-build-failed-for-goos-linux-arm/9267

Try sending them a tiny PR for this that just makes the constant linked in that issue an `int64` and see what they think.

It looks like they may have other x86 problems (https://discuss.dgraph.io/t/fix-badger-builds-on-32-bit-machines/8679) but at least this won't block us on compiling.